README.md

```diff
@@ -70,7 +70,22 @@ Contributed by: [@seunfashina002](https://github.com/seunfashina002)
   
 
 >  Ullam rerum but as we dey believe, laudantium commodi we no go just sit down tempore odio cum dey wait make manna fall from heaven fugiat provident, make we try dey show love sint velit aliquid, make we dey helep people non autem quod nesciunt. if you no get work voluptates veritatis find work inventore, no dey thief person money voluptatem commodi con dey call am grace amet ut rem omnis. Nigeria na we own oo, quia perspiciatis.
- 
+
+
+## Pidsum Eiii Brother Bernard
+
+Contributed by: [@seunfashina002](https://github.com/seunfashina002)
+
+> Tori tori o Magni fugiat qui I don carry another gist come praesentium hic maxime. If you no dey live under rock Id asperiores provident you go don see video wey people dey yarn eiii Brother Bernard sit inventore debitis. I no say una no sabi how he take start facere aut quibusdam, no wahala I go run am for you like Usain bolt soluta dignissimos beatae
+
+
+> voluptatem commodi the tori be say, one woman wey go funeral for church  excepturi repudiandae fall yakata for inside grave wey dem wan bury Reverend Father Bernard Kofi Ackon voluptates Perspiciatis wey die for cape coast hosipital. 
```

Comment:
Tori tori o! Magni fugiat qui I don carry another beta gist come praesentium hic maxime. If you no dey live under rock, Id asperiores provident, you go don see video wey people dey yarn "Eiii Brother Bernard" sit inventore debitis. I know say una no sabi how e take start, facere aut quibusdam, no wahala I go run am for you like Usain Bolt soluta dignissimos beatae.

> Voluptatem commodi, the tori be say, one woman wey go birthday party for church excepturi repudiandae, she slip enter big bowl of chocolate wey dem prepare for Brother Bernard Kofi Ackon im birthday celebration. Voluptates Perspiciatis, na so e be for the party wey dem dey celebrate for cape coast area.


You can consider the above as a semantic improvement to the story, and instead of a hospital and funeral, we use a birthday with a pot of chocolate